src/components/skeletons/FilterSkeleton/filter-skeleton.module.scss

```
@@ -0,0 +1,53 @@
+@import "src/styles/scaffold";
+
+.fs-filter-skeleton {
+  // --------------------------------------------------------
+  // Design Tokens for Filter Skeleton
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-filter-skeleton-margin-top            : var(--fs-spacing-1);
+
+  // Title
+  --fs-filter-skeleton-title-max-width       : 30%;
+  --fs-filter-skeleton-title-margin-bottom   : var(--fs-spacing-2);
+
+  // Content
+  --fs-filter-skeleton-content-min-height    : var(--fs-spacing-8);
+  --fs-filter-skeleton-content-margin-bottom : var(--fs-spacing-0);
+  --fs-filter-skeleton-content-padding       : var(--fs-spacing-1) var(--fs-spacing-1) var(--fs-spacing-0);
+  --fs-filter-skeleton-content-border-color  : var(--fs-border-color-light);
+  --fs-filter-skeleton-content-border-width  : var(--fs-border-width);
+  --fs-filter-skeleton-content-border-radius : var(--fs-border-radius);
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  margin-top: var(--fs-filter-skeleton-margin-top);
+
+  @include media("<notebook") { display: none; }
+
+  [data-fs-skeleton-variant="text"] {
+    max-width: var(--fs-filter-skeleton-title-max-width);
+    margin-bottom: var(--fs-filter-skeleton-title-margin-bottom);
+  }
+
+  [data-fs-filter-skeleton-content] {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    padding: var(--fs-filter-skeleton-content-padding);
+    overflow: hidden;
+    border-color: var(--fs-filter-skeleton-content-border-color);
+    border-style: style;
```

```suggestion
    border-style: solid;
```

or maybe group all props.
```css
border: var(--fs-filter-skeleton-content-border-width) solid var(--fs-filter-skeleton-content-border-color);
```